docker/test/fasttest/Dockerfile

```diff
@@ -24,7 +24,7 @@ RUN apt-get update \
 
 RUN pip3 install numpy scipy pandas Jinja2
 
-ARG odbc_driver_url="https://github.com/ClickHouse/clickhouse-odbc/releases/download/v1.1.4.20200302/clickhouse-odbc-1.1.4-Linux.tar.gz"
+ARG odbc_driver_url="https://github.com/ClickHouse/clickhouse-odbc/releases/download/v1.1.10.20210822/clickhouse-odbc-1.1.10-Linux.tar.gz"
```

Comment:
In tests I see `Can't open lib '/usr/local/lib64/libclickhouseodbcw.so'`

Interesting what've changed...

Reply:
Configuration changed with the new version, there were relative paths to the lib, now there is `/usr/local/lib64`, which doesn't even exist in the `fasttest` image. I decided that I don't want to update `clickhouse-odbc` version, and that the URI that it uses (`?default_format=...`, notice missing `/` in the beginning) is valid.